Task2/MiddleCharacter.js

```diff
@@ -0,0 +1,13 @@
+//2- http://www.codewars.com/kata/get-the-middle-character
+function getMiddle(s) {
+    var len = s.length;

```

Comment:
Use meaning full names for variables, to increase the readability of your code
don't use 's'.
use 'stringLength' instead of 'len'